responses/fr/HassTurnOn.yaml

```diff
@@ -6,6 +6,6 @@ responses:
       fans: "Ventilateurs allumés"
       cover: "Ouvert"
       lock: "Fermé"
-      scene: "Activée"
-      script: "Démarré"
+      scene: "Scène activée"
+      script: "Script exécuté"
```

Comment:
I would still keep this feeling of something that is starting but not done.
A script can take time to complete, you can have delays, or even pause until a specific trigger is matched.

I just tried with a simple example:
```yaml
alias: Test script
sequence:
  - delay:
      hours: 0
      minutes: 0
      seconds: 10
      milliseconds: 0
  - service: light.toggle
    metadata: {}
    data: {}
    target:
      entity_id: light.bureau
```

When I say "Lance test script", the response is immediate (Today `Démarré`), but my office light is toggled after some delay, 10 seconds.

Here are a few proposals from me
- Script démarré
- Script lancé

Reply:
It makes sense, I thought it was the same as automation with custom responses e.g. triggered after the script completion.